### PR TITLE
Update C# change notes for 1.21 to mention two new queries

### PR DESCRIPTION
**Pull request by felicity-semmle**
@calumgrant - I've taken the information in the *Purpose* column from the query descriptions, but shortened them a little. I hope that I've not changed the meaning.

### CODE REVIEW

File: change-notes/1.21/analysis-csharp.md [hunk 1]
```diff
@@ -4,11 +4,17 @@
...
 
+## New queries
+
+| **Query**                                     | **Tags**                                             | **Purpose**                                                                                                                                                                 |
+|-----------------------------------------------|------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Class defines a field that uses an ICryptoTransform class in a way that would be unsafe for concurrent threads (`cs/thread-unsafe-icryptotransform-field-in-class`) | concurrency, security, external/cwe/cwe-362 | Highlights classes with fields that make use of a static `System.Security.Cryptography.ICryptoTransform` object. Using these classes in concurrent threads is dangerous. It may result an error and may also give incorrect results. |
```
Comment (hvitved) [inline change-notes/1.21/analysis-csharp.md]: result in

File: change-notes/1.21/analysis-csharp.md [hunk 2]
```diff
@@ -4,11 +4,17 @@
...
+| Potential usage of an object implementing ICryptoTransform class in a way that would be unsafe for concurrent threads (`cs/thread-unsafe-icryptotransform-captured-in-lambda`) | concurrency, security, external/cwe/cwe-362 | Highlights instances of classes where a field of type `System.Security.Cryptography.ICryptoTransform` is captured by a lambda, and appears to be used in a thread initialization method. |
+
```
Comment (hvitved) [inline change-notes/1.21/analysis-csharp.md]: I hadn't noticed before now, but the query names are quite verbose. I would prefer something like

`Class defines a field that uses an ICryptoTransform class in a way that would be unsafe for concurrent threads` ->
`Thread-unsafe use of a static ICryptoTransform field`

and

`Potential usage of an object implementing ICryptoTransform class in a way that would be unsafe for concurrent threads` -> `Thread-unsafe capturing of an ICryptoTransform object`

If you agree, could we update the query names on this PR as well?

**Comment by felicity-semmle** [comment]
Apologies. Managed to get my local branch very confused due to Windows case insensitivity. Am almost straight and will address your first comment now.

File: change-notes/1.21/analysis-csharp.md (hunk 2 above)
Comment (felicity-semmle) [reply change-notes/1.21/analysis-csharp.md]: As discussed separately, this change would be better made on `master`.

**Comment by felicity-semmle** [comment]
Thanks for the review @hvitved. Feedback addressed.

**Pull request merged**